hack/make-rules/build_with_container.sh

```diff
@@ -24,11 +24,14 @@ KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 MOUNTPATH="${MOUNTPATH:-/kubeedge}"
 KUBEEDGE_BUILD_IMAGE=${KUBEEDGE_BUILD_IMAGE:-"kubeedge/build-tools"}
 DOCKER_GID="${DOCKER_GID:-$(grep '^docker:' /etc/group | cut -f3 -d:)}"
+# please edit ENV_LIST to support more env variables
+ENV_LIST="${ENV_LIST:-ARM_VERSION}"
```

Comment:
```suggestion
ENV_LIST=${ENV_LIST:-"ARM_VERSION"}
```
This will be more readable?

Reply:
if we want to pass in two environment or more, the before format can specify the needed double quotation, just like `env | grep "ARM_VERSION\|ANOTHER_ENV"` but not `env | grep ARM_VERSION\|ANOTHER_ENV` without quotations, the latter can not get right env.

Reply:
Just have a check, seems have no difference?
```
➜  ~ ENV_LIST="${ENV_LIST:-GVM_ROOT\|GVM_VERSION}"
➜  ~ env | grep ${ENV_LIST}                       
GVM_ROOT=/root/.gvm
GVM_VERSION=1.0.22
➜  ~ ENV_LIST=${ENV_LIST:-"GVM_ROOT\|GVM_VERSION"}
➜  ~ env | grep ${ENV_LIST}                       
GVM_ROOT=/root/.gvm
GVM_VERSION=1.0.22
```
I think here ARM_VERSION is used as a string instead of env value, so with quotation seems more easy to understand :)

Reply:
yes, we must ensure the value is quoted string, but when users set ENV_LIST by themselves, they may forget the quotation just like `export ENV_LIST=ARM_VERSION\|TEST`, so make release may fail. the former adds quotation at the outermost layer, so it can ensure the ENV_LIST environment value is a quoted string.

Reply:
OK, and seems if you use `ARCH` in https://github.com/kubeedge/kubeedge/blob/master/hack/make-rules/release.sh#L60, the env value `ARM_VERSION` will not needed.

Reply:
fixed